lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -225,7 +225,7 @@ def self.ruby_method_runnable?(aem)
 
     def self.setup_drb_for_ruby_method
       require 'drb/timeridconv'
-      DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
+      @@global_id_conv = DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
       drb_front  = MiqAeMethodService::MiqAeServiceFront.new
       drb        = DRb.start_service("druby://127.0.0.1:0", drb_front)
```

Comment:
@mkanoor the timer thread keeps references to all drb objects and rotates those objects out if they weren't accessed every timeout seconds so it can be GC'd.  But yes, there should be a better way.